src/reducers/popup.ts

```diff
@@ -66,6 +66,11 @@ export default (state: any = defaultState, action: any) => {
         ...state,
         values: [...state.values, ...action.variables.map((variable: string) => ({ name: variable, value: null }))],
       }
+    case SELECT_METHOD:
+      if (action.method === 'function') {
```

Comment:
This will probably need to be updated once Bill's custom functions PR is merged. Would it work to check for _not_ custom or seedzone method?

Reply:
That would work. Changed that.